### PR TITLE
README: Update general documentations and mention some WIP

### PR DESCRIPTION
**Pull request by xescugc**
Also update 'json' for 'tfstate' when mentioning it, and added a minimal version of GO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +81,8 @@ apply specific provider logic if supported. If not supported, then basic graph i
 
 ### AWS
 
+WIP: Grouping and IAM connections
```
Comment (xlr-8) [inline README.md:84]: I'd rather add a note or have it displayed differently:
* X: OK
* Y: OK
* Z: WIP

Plus IAM is pretty "vague" there are like 20 resources related to IAM. Building the graph based on the IAM, will require to build a tree of IAM itself to know who can access what basically. Perhaps we know already the list?

Comment (tormath1) [reply README.md:84]: bonus: we can add the link of the issue https://github.com/cycloidio/inframap/issues/6 for "grouping"

Comment (xescugc) [reply README.md:84]: Which names would you use then?

**Comment by xescugc** [comment]
Done

**Pull request merged**